src/components/learner-credit-management/BudgetCard.jsx

```diff
@@ -1,80 +1,102 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useOfferSummary } from './data';
+import { useSubsidySummaryAnalyticsApi } from './data';
 import SubBudgetCard from './SubBudgetCard';
 import { BUDGET_TYPES } from '../EnterpriseApp/data/constants';
 
 /**
- * Renders one or more budget cards for the given offer (enterprise or Subsidy from enterprise-subsidy). If the offer is
- * an enterprise offer, it will render a single card. If the offer is a Subsidy, it will render one card for
- * each associated budget.
+ * Renders one or more budget cards for the given budget. If the budget is
+ * an enterprise offer, it will render a single card. If the budget is a Subsidy,
+ * it will render one card for each associated budget. If the budget is a Policy,
+ * it will also render a single card.
+ *
+ * @param {Object} budget Represents either:
+ *  - Enterprise Offer
```

Comment:
[nit] I would append the API here as well  to be consistent similar to how you appended 
it to the comments below about Subsidy and Policy